modular_skyrat/modules/implants/code/medical_designs.dm

```diff
@@ -11,6 +11,84 @@
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_SECURITY
 
+/datum/design/cyberimp_razorwire
+	name = "Razorwire Spool Implant"
+	desc = "A long length of cutting wire so impossibly thin that it causes grevious wounds in anything you slash with it. \
+		Its long enough that you'd probably be able to hit someone with it from a little further away than normal."
+	id = "combat_implant_razorwire"
+	build_type = MECHFAB
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 2,
+		/datum/material/glass = HALF_SHEET_MATERIAL_AMOUNT,
+		/datum/material/titanium = SHEET_MATERIAL_AMOUNT * 3,
+	)
+	construction_time = 30 SECONDS
+	build_path = /obj/item/organ/internal/cyberimp/arm/razorwire
+	category = list(
+		RND_CATEGORY_CYBERNETICS + RND_SUBCATEGORY_CYBERNETICS_IMPLANTS_COMBAT
+	)
+	departmental_flags = DEPARTMENT_BITFLAG_SECURITY
+
+/datum/design/cyberimp_shell_launcher
+	name = "Shell Launch System Implant"
+	desc = "A complex housing for implanting a shell launch system into an arm. Holds a single shot barrel that can hold either twelve gauge or .980 Tydhouer shells."
+	id = "combat_implant_shell_launcher"
+	build_type = MECHFAB
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 2,
+		/datum/material/glass = HALF_SHEET_MATERIAL_AMOUNT,
+		/datum/material/titanium = SHEET_MATERIAL_AMOUNT * 3,
```

Comment:
It feels a little cheap to me, like, I feel there probably would be something more exotic also mixed into this, not sure what though.